kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/parquet/ParquetFileWriterSuite.scala

```diff
@@ -167,6 +168,20 @@ class ParquetFileWriterSuite extends AnyFunSuite
             Seq.empty, tableSchema(goldenTablePath("decimal-various-scale-precision"))),
           29 // how many columns have the stats collected from given list above
         )
+    },
+    // Read a iceberg compat v2 data with field ids and nested ids and write it back
+    Seq((200, 1)).map {
+      case (targetFileSize, expParquetFileCount) =>
+        (
+          "write iceberg compat v2 data with field ids (no stats)", // test name
+          "table-with-columnmapping-mode-id",
+          targetFileSize,
+          expParquetFileCount,
+          6, /* expected number of rows written to Parquet files */
```

Comment:
where does this magic number come from? what's the relationship between a targetFileSize of 200 and 6 output rows?